inject-generator/pom.xml

```diff
@@ -37,12 +37,6 @@
       <scope>provided</scope>
     </dependency>
 
-    <dependency>
```

Comment:
Looks unrelated?

Reply:
Hmm, thought it came about when I pulled from master and had conflicts